Java/jni/native_c4database.cc

```diff
@@ -82,11 +82,11 @@ Java_com_couchbase_litecore_C4Database_copy(JNIEnv *env, jclass clazz,
 /*
  * Class:     com_couchbase_litecore_C4Database
  * Method:    free
- * Signature: (J)Z
+ * Signature: (J)
```

Comment:
1. Please fix the signature [here](https://github.com/couchbase/couchbase-lite-core/blob/master/Java/src/com/couchbase/litecore/C4Database.java#L339) to return void as well.
2. To return void, the Signature should be `(J)V`